### ISSUE TITLE
[Accessibility]: Text "unknown" for certain failed LSIF uploads is unclear

### ISSUE THREAD

**Issue by Strum355** [sourcegraph/sourcegraph#35329]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33512

### Problem description

In certain circumstances, an uploads repository name cannot be displayed, for which we fallback to the text  'unknown'. This is not clear to a screen reader _what_ is unknown.

### Expected behavior

Replace the text or give a better aria-label replacement

### Additional details

![image](https://user-images.githubusercontent.com/18282288/167939618-8b12f7e7-230d-4cb8-a5b3-0bcb97eaed92.png)

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
